extensions/amp-story/1.0/amp-story.js

```diff
@@ -939,14 +946,15 @@ export class AmpStory extends AMP.BaseElement {
       return;
     }
 
-    switch (e.keyCode) {
-      // TODO(newmuis): This will need to be flipped for RTL.
-      case KeyCodes.LEFT_ARROW:
-        this.previous_();
-        break;
-      case KeyCodes.RIGHT_ARROW:
-        this.next_();
-        break;
+    const keyDirection = e.keyCode;
+    const isRtl = this.storeService_.get(StateProperty.IS_RTL_STATE);
```

Comment:
Nit: to further avoid confusions between `isRTL` and `isRtl`, maybe `RTLState`?

Reply:
Technically, I think it's supposed to be `Rtl` per the style guide (not that we strictly follow the style guide), since only the first letter of acronyms get capitalized.

Reply:
Per [styleguide](https://engdoc.corp.google.com/eng/doc/devguide/js/styleguide.md#naming-camel-case-defined) I will stick to `RtlState`